src/components/PeopleTableBody/PeopleTableBody.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  person: Person;
+  selectedPersonSlug: string;
+}
+
+export const PeopleTableBody: FC<Props> = ({
+  person,
+  selectedPersonSlug,
+}) => {
+  const {
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.